flexibleengine/resource_flexibleengine_dli_queues_v1.go

```diff
@@ -0,0 +1,244 @@
+package flexibleengine
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+
+	"github.com/huaweicloud/golangsdk/openstack/common/tags"
+	"github.com/huaweicloud/golangsdk/openstack/dli/v1/queues"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+var regexp4Name = regexp.MustCompile(`^[a-z0-9_]+$`)
+
+func ResourceDliQueueV1() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDliQueueCreate,
+		Read:   resourceDliQueueV1Read,
+		Delete: resourceDliQueueV1Delete,
+
+		Schema: map[string]*schema.Schema{
+			"queue_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
+					v := val.(string)
+					if !regexp4Name.MatchString(v) {
+						errs = append(errs, fmt.Errorf("%q can contain only digits, lower letters, and underscores (_) ", key))
+					}
+					return
+				},
+			},
+
+			"queue_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  "sql",
+			},
+
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"cu_count": {
```

Comment:
- It's better to name `cu_specifications` , same as the console. 
- Only 16, 64, 256 and 512 are supported, advised to add validate function, e.g.: `ValidateFunc: validate.IntInSlice([]int {16, 64, 256, 512})`.

Reply:
1. name is same with the fieldname in api
2. OK， add validatefunc